scrapers/TransSensual.yml

```diff
@@ -0,0 +1,63 @@
+name: "TransSensual"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - transsensual.com/scene
+    scraper: sceneScraper
+    
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - transsensual.com/movie
+    scraper: movieScraper
+    
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="wxt7nk-4 fSsARZ"]/text()
+# do /html/head/title if you want title with "TS Porn | Transsensual" added after it.
+      Details: //span/div[contains(.,"Description")]/../div[not(contains(.,"Description"))]/text()
+      Date:
+        selector: //span/div[contains(.,"Release Date")]/../text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Performers:
+        Name: //div[@class="wxt7nk-5 cWGMuL"]/span/a/text()
+      Studio:
+        Name: //meta[@name="dti.network"]/@content
+      Tags:
+        Name:
+          selector: //span/div[contains(.,"Categories")]/../a
+          postProcess:
+            - replace:
+              - regex: ","
+                with: ""
+      Image:
+        selector: //img[contains(@class, "lSdde")]/@src
+        
+  # The movie section is kinda weird. The movies seems like they are made after the scenes.
+  # e.g. they don't seem to have front/back covers, and are basically just a compilation
+  # of some scenes they released with same theme? 
+  # So a movie has date as the earliest released scene, and other scenes have their own.
+  # Description is just name of the 'movie', so gonna go with whole 'desc & cat' as synopsis.
+  # So i'd say movies on TransSensual aren't 'proper'. But here goes:
+  movieScraper:
+    movie:
+      Name: //h2[@class='wxt7nk-4 fSsARZ']/text()
+#      Director: 
+#      Duration: 
+      Aliases: 
```

Comment:
Instead of aliases it gets the performers. Is this intentional?

Reply:
Yes, that was intentional.
Some movies got title-aliases?
Just liked having names in that field myself, but we can change it.
I guess with a movie section overhaul, some of these things will change anyways?